redhat-developer/intellij-openshift-connector
# fix: can't connect to a cluster with private certificate

## What is the purpose of this change? What does it change?
Openshift view can't list / createcompoennts on clusters using private certificates.

## Was the change discussed in an issue?
fixes #547

## How to test changes?
follow https://github.com/redhat-developer/intellij-kubernetes/issues/600#issuecomment-1562742941 to set up a cluster with private certificate ( CRC ), then please also modify the kubeconfig file to something like
```
- cluster:
    insecure-skip-tls-verify: true
    server: "https://api.crc.testing:6443"
  name: "api-crc-testing:6443"
```
without this manual change, odo can't access the cluster.
